### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by alex-phillips**

### CODE REVIEW

**Review by Roxedus — APPROVED** [review]

File: jenkins-vars.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+
+# jenkins variables
+project_name: docker-paperless-ngx
+external_type: github_stable
+release_type: stable
+release_tag: latest
+ls_branch: master
```
Comment (aptalca) [inline jenkins-vars.yml]: This needs to be `main`

**Review by aptalca — CHANGES_REQUESTED** [review]
See comment above

**Review by aptalca — DISMISSED** [review]

**Comment by aptalca** [comment]
Both arm logs have various errors

**Comment by alex-phillips** [comment]
> Both arm logs have various errors

@aptalca could be a qemu issue. This same error existed in the old NG build logs and rox has confirmed on his hardware that it is working properly.

**Comment by darkmattercoder** [comment]
I built it on my M1 using aarch64 and can confirm, that I do not see those errors. I am still trying to verify it on my raspberrypi for armhf.

**Review by aptalca — APPROVED** [review]

**Pull request merged**